sympy/matrices/common.py

```diff
@@ -2845,7 +2845,7 @@ def __sub__(self, a):
 
     @call_highest_priority('__rtruediv__')
     def __truediv__(self, other):
-        return self.__div__(other)
+        return self.__truediv__(other)
```

Comment:
Is this not an infinite recursion?

Reply:
Yes, this looks like an error. The second commit was done with a find and replace, was waiting for CI to smoke out the bugs